cadt-ui/templates/cadt.local.conf.j2

```diff
@@ -0,0 +1,57 @@
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name {{ certbot_domains | join( " " ) }};
+
+    root /dev/null;
+
+    include /etc/nginx/snippets/lets-encrypt-webroot.conf;
+
+    location / {
+        rewrite (.*) {{ cadt_ui_host }}$1 permanent;
+    }
+}
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    root /var/www/{{ cadt_ui_host }}/build;
+
+    server_name {{ certbot_domains | join( " " ) }};
+
+    ssl_certificate /etc/letsencrypt/live/{{ certbot_domains | first }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ certbot_domains | first }}/privkey.pem;
+
+    ssl_protocols             TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers               ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+    ssl_buffer_size           8k;
+    ssl_session_cache         shared:SSL:50m;
+    ssl_session_timeout       1d;
+    ssl_session_tickets       off;
+
+    resolver                  8.8.8.8;
```

Comment:
I'm inclined to say use 1.1.1.1 for this, since our dns is on cloudflare, but not a blocker

Reply:
updated to 1.1.1.1